plugins/documentation/docs/de_de/module-simple.md

```diff
@@ -2,171 +2,60 @@
-                          <p class="search_it-teaser">'.$hit['highlightedtext'].'</p>
-                      </li>'; 
-            } else {                                   
-                
-                
-                echo '<p class="search_it-missing_type">Das Suchergebnis vom Typ <i class="search_it-type">'.$hit['type'].' </i> kann nicht dargestellt werden.</p>';
-            }
-
-        } 
-        echo '</ul>';
-    } else if(!$result['count']) { 
-        echo '<p class="search_it-zero">Die Suche nach <i class="search_it-request">'.rex_escape($request).' </i> ergab keine Treffer.</p>';
-    }
-} 
-    ?>
-</section>
-```
-
-## Modulausgabe (mit Erläuterungen)
-
-```
-<section class="search_it-modul">
-    <p class="search_it-demotitle">[search_it] Suchergebnisse - Einfaches Beispielmodul</p>
-    <?php
-$server = rtrim(rex::getServer(),"/"); // Aktuelle Website-Adresse ohne Slash am Ende;
-$request = rex_request('search', 'string', false); // GET/POST-Anfrage: Casting als String
+<?php
+$article_id = rex_article::getCurrentId();
+$article = rex_article::get($article_id);
+$request = rex_request('search', 'string', false);
 
 if($request) { // Wenn ein Suchbegriff eingegeben wurde
-    $search_it = new search_it(); // Suche initialisieren
-    $result = $search_it->search($request); // Suche ausführen
-	# dump($result); // Zum Debuggen ausgeben.
-
-    if($result['count']) { // Wenn Ergebnisse vorhanden sind...
-        echo '<h2 class="search_it-headline">{{ Suchergebnisse }}</h2>'; // Sprog-AddOn zur Übersetzung benutzen
-
-        echo '<ul class="search_it-results">';                           
-        foreach($result['hits'] as $hit) { // Jeder Treffer in $hit
-
-            # dump($hit); // Zum Debuggen ausgeben.
-            if($hit['type'] == 'article') { // Wenn der Treffer aus der Artikelsuche stammt 
-                $article = rex_article::get($hit['fid']); // REDAXO-Artikel-Objekt holen
-
-				// falls YRewrite genutzt wird
-				$hit_server = $server;
-				if(rex_addon::get('yrewrite')->isAvailable()) {
-					$hit_domain = rex_yrewrite::getDomainByArticleId($hit['fid'], $hit['clang']);
-					$hit_server = rtrim($hit_domain->getUrl(), "/");
+	$server = rtrim(rex::getServer(), "/");
+	
+	print '<section class="search_it-hits">';
+	
+	// Suche initialisieren (nur Artikel in der aktuellen Sprache)
+    $search_it = new search_it(rex_clang::getCurrentId());
+	// Suche ausführen
+    $result = $search_it->search($request);
+
+	echo '<h2 class="search_it-headline">Suchergebnisse</h2>';
+	if($result['count']) {
+		// Suchergebnisse ausgeben
+		echo '<ul class="search_it-results">';                           
+        foreach($result['hits'] as $hit) {
+            if($hit['type'] == 'article') {
+				// Artikel
+                $article_hit = rex_article::get($hit['fid']);
+				// Falls YCom Addon genutzt wird: zuerst Benuterrechte prüfen
```

Comment:
Was zur Hölle hat denn YCom in einem "module-SIMPLE" zu suchen?

Reply:
Definitiv nichts. Ist entfernt.